lib/streamingly/serde.rb

```
@@ -34,6 +34,7 @@ def self.from_csv(string)
 
     def self.from_tabbed_csv(string)
       k, v = string.split("\t", 2)
+      return if k.nil? || v.nil?
       key = from_string_or_csv(k)
```

I like the idea of raising an exception rather than silently failing